source/material_model/equation_of_state/interface.cc

```diff
@@ -93,11 +73,6 @@ namespace aspect
   {
 #define INSTANTIATE(dim) \
   template struct EquationOfStateOutputs<dim>; \
-  template void fill_averaged_equation_of_state_outputs<dim> (const EquationOfStateOutputs<dim> &, \
-                                                              const std::vector<double> &, \
-                                                              const std::vector<double> &, \
-                                                              const unsigned int, \
-                                                              MaterialModelOutputs<dim> &); \
```

Comment:
You have removed the explicit instantiation here, but have not added it into utilities.cc that is why the unity tester failed.